src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/AnnotationLocationCheck.java

```diff
@@ -262,33 +266,35 @@ public final void setAllowSamelineMultipleAnnotations(boolean allow) {
 
     @Override
     public void visitToken(DetailAST ast) {
-        final DetailAST modifiersNode = ast.findFirstToken(TokenTypes.MODIFIERS);
-
-        if (modifiersNode != null) {
-            checkAnnotations(modifiersNode, getExpectedAnnotationIndentation(modifiersNode));
+        DetailAST annotationsNode = ast.findFirstToken(TokenTypes.MODIFIERS);
```

Comment:
not sure this is good update to rename variable.
we searched for modifiers, and we should found modifiers node.
it is not good to reuse variable.

Reply:
done